tests/improver-threshold/01-help.bats

```diff
@@ -91,6 +92,8 @@ optional arguments:
   --collapse-coord COLLAPSE-COORD
                         An optional ability to set which coordinate we want to
                         collapse over. The default is set to None.
+  --vicinity VICINITY   If set, distance in metres used to define the vicinity
+                        within which to search for an occurrence.
```

Comment:
I wonder if we could elaborate here now - maybe `within which to search for a maximum probability`?